.github/workflows/goreleaser.yml

```diff
@@ -121,4 +121,5 @@ jobs:
       registry-username: ${{ github.actor }}
       digest: ${{ needs.goreleaser.outputs[format('digest-{0}', matrix.target)] }}
     secrets:
-      registry-password: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+      registry-password: ${{ secrets.GITHUB_TOKEN }}
+      
```

Comment:
FYI: the comment on this commit says "removed newline" but what I see here is adding two newlines.

<img width="216" alt="image" src="https://github.com/user-attachments/assets/fe86bbe8-f5e3-449c-87ab-45cf5dc629dd" />